apps/api/src/app/environments/environment.ts

```diff
@@ -92,6 +92,14 @@ const devConfig = {
       process.env.REGULATIONS_API_URL ??
       'https://reglugerdir-api.herokuapp.com/api/v1',
   },
+  fjarmalDomain: {
+    username: process.env.FINANCE_USER ?? '',
+    password: process.env.FINANCE_PASS ?? '',
+    xroadApiPath:
+      process.env.XROAD_FINANCES_PATH ??
+      'IS-DEV/GOV/10021/FJS-Public/financeIsland',
+    ttl: parseInt(process.env.FJARMAL_TTL, 10) || 600,
```

Comment:
No action required but more of mental note to self that we want to prevent having these inline defaults and avoid working with the defaults in the cluster when the env vars get forgotten to be added to the helm config